source/app/components/pages/dashboard/index.tsx

```diff
@@ -1,19 +1,60 @@
+/* global HTMLInputElement */
 // packages
-import React, { useEffect, useContext, useState } from 'react';
-import Grid from '@mui/material/Grid';
 import Divider from '@mui/material/Divider';
+import Grid from '@mui/material/Grid';
+import React, { useEffect, useContext, useState } from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+import InputBase from '@mui/material/InputBase';
 import Typography from '@mui/material/Typography';
-import TemplateList from '../../organisms/template-list';
+import { styled } from '@mui/material/styles';
 
 // scripts
 import SettingsButton from '../../molecules/settings-section';
+import TemplateList from '../../organisms/template-list';
 import styles from './styles';
 import { GIT_URL } from '../../../utils/regex';
 import { GlobalStateContext } from '../../../context/MessageContext';
 import { IFolder } from '../../../utils/interfaces/remoteFolders.interface';
 import { remoteList } from '../../../api/remote-list';
 
```

Comment:
Extract styles from index file, left only logic and html tags. Create file of styles